tests/test_atom_mapping.py

```diff
@@ -90,12 +92,13 @@ def test_complete_rings_only():
         mol_b,
         ring_cutoff=0.1,
         chain_cutoff=0.2,
-        max_visits=1e7,  # 10 million max nodes to visit
+        max_visits=100,  # 100 max nodes to visit
```

Comment:
Previously with this value set to 100 this would raise an exception, despite finding the expected mapping.

Reply:
Does it make sense to test both 100, 1e7? Not sure how many nodes of the search tree are actually visited in this case.

Reply:
I should make this a separate test, was just the quick case I came up with demonstrating a change in max visits failed without the change of returning a mapping even if things had timed out